jekyll/_cci2/env-vars.md

```diff
@@ -16,19 +16,40 @@ This document describes using environment variables in CircleCI in the following
 
 ## Overview
 
-Environment variables are used according to a specific precedence order, as follows:
```

Comment:
Please leave this section as-is, we had it general to specific before and changed it to specific to general in March based on the feedback from customers on Contexts. (https://github.com/circleci/circleci-docs/commit/34af95df586700293547a7aba0843da65ef3737f#diff-c9b824d50d2de38d360f6856babb1c4f)

Reply:
Hmm, this linked diff doesn't point to a general --> specific Overview. I can definitely see the argument for a specific --> general; that's how [Wikipedia](https://en.wikipedia.org/wiki/Scope_(computer_science)#Levels_of_scope) does it.

Regardless of the order used, though, I think the ordering of the sections in the document should match the precedence order used in the Overview.